modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,488 @@
+    "artifactregistry.versions.list",
+    "logging.logEntries.create",
+    "logging.logEntries.route",
+    "monitoring.metricDescriptors.create",
+    "monitoring.metricDescriptors.get",
+    "monitoring.metricDescriptors.list",
+    "monitoring.monitoredResourceDescriptors.get",
+    "monitoring.monitoredResourceDescriptors.list",
+    "monitoring.timeSeries.create",
+    "cloudnotifications.activities.list",
+    "monitoring.alertPolicies.get",
+    "monitoring.alertPolicies.list",
+    "monitoring.dashboards.get",
+    "monitoring.dashboards.list",
+    "monitoring.groups.get",
+    "monitoring.groups.list",
+    "monitoring.notificationChannelDescriptors.get",
+    "monitoring.notificationChannelDescriptors.list",
+    "monitoring.notificationChannels.get",
+    "monitoring.notificationChannels.list",
+    "monitoring.publicWidgets.get",
+    "monitoring.publicWidgets.list",
+    "monitoring.services.get",
+    "monitoring.services.list",
+    "monitoring.slos.get",
+    "monitoring.slos.list",
+    "monitoring.snoozes.get",
+    "monitoring.snoozes.list",
+    "monitoring.timeSeries.list",
+    "monitoring.uptimeCheckConfigs.get",
+    "monitoring.uptimeCheckConfigs.list",
+    "opsconfigmonitoring.resourceMetadata.list",
+    "resourcemanager.projects.get",
+    "stackdriver.projects.get",
+    "stackdriver.resourceMetadata.list",
+    "dns.changes.create",
+    "dns.changes.get",
+    "dns.changes.list",
+    "dns.managedZones.list",
+    "dns.resourceRecordSets.create",
+    "dns.resourceRecordSets.delete",
+    "dns.resourceRecordSets.get",
+    "dns.resourceRecordSets.list",
+    "dns.resourceRecordSets.update",
+    "secretmanager.versions.access",
+    "stackdriver.resourceMetadata.write",
+    "storage.objects.get",
+    "storage.objects.list"
+  ],
+}
+EOT
+
+gcloud iam roles create redpanda_gke_utility_role --project=<service-project-id> --file redpanda-gke.role
+
+gcloud projects add-iam-policy-binding <service-project-id> \
+  --member="serviceAccount:redpanda-gke@<service-project-id>.iam.gserviceaccount.com" \
+  --role='projects/<service-project-id>/roles/redpanda_gke_utility_role'
+```
+====
+
+* Bind GKE service account with the Workload Identity role
++
+.Show commands
+[%collapsible]
+====
+```bash
+gcloud iam service-accounts add-iam-policy-binding redpanda-gke@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[cert-manager/cert-manager]"
+gcloud iam service-accounts add-iam-policy-binding redpanda-gke@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[external-dns/external-dns]"
+```
+====
+
+. In the https://cloudv2.redpanda.com[Redpanda Cloud UI^], start to xref:./create-byoc-cluster-gcp.adoc[create a BYOC cluster]. This generates a Redpanda ID, which is necessary to create certain resources.
```

Comment:
I'm a wee bit confused by the wording "start to create a BYOC cluster" here...so do I follow through and create it? Or only go so forward with this until I can grab the Redpanda ID?

Reply:
I think this text does not clearly describe a "step". Maybe:
 In the https://cloudv2.redpanda.com[Redpanda Cloud UI^], start to xref:./create-byoc-cluster-gcp.adoc[create a BYOC cluster] so that you can get the Redpanda ID, which is require here.